extensions/amp-viewer-assistance/amp-viewer-assistance.md

```diff
@@ -0,0 +1,20 @@
+# amp-viewer-assistance
+
+The amp-viewer-assistance element provides specification of AMP viewer
+configuration information and invocation of AMP viewer assisted behavior.
```

Comment:
This documentation needs some love. :) 

We ought to:
1. Clearly describe the high-level product use case. E.g. this is meant for "assistive behaviors on AMP pages, facilitated by the viewer."
2. Document AMP page/viewer API contract e.g. "requestSignIn". 
3. Document element actions like "signIn" and "updateActionState". 
4. Follow the format from other extension documentation (see amp-access.md).

/cc @CrystalOnScript

Reply:
Agreed! More documentation will be committed to this PR soon.

P.S. That is a very nice way of requesting for more documentation! :+1: :smile:

Reply:
Just updated the documentation to address some of those comments.